### DISCUSSION TITLE
Anyone working on support for craftbot+ printer?

### DISCUSSION THREAD

**Discussion by chaikk** [supermerill/SuperSlicer#1905] [Q&A]
Can't get it to work with craftbot+ printer, any work around? look like gode generated not compatible with printer firmware.

**Comment by supermerill** [comment]
what is the firmware?

**Reply by chaikk** [reply]
Got this from printer screen 

Model: Craftbot
HMI version: 1.1.18142/2 - 2019-02-18
pr3Dator version: 1.1.12360/4 - 2019-02-18

hope this help thanks
